strings/README.md

```diff
@@ -18,3 +18,16 @@ You could use any online IDE (for an example [.net Finddle](https://dotnetfiddle
 
 1. [Palindrome Check](java/palindrome.java)
 2. [All subsequences](java/sequence.java)
+
+# String algorithms
```

Comment:
actually, this title looks better than `String operations` 

here is the proposed change 

change the title from `#String operations` to `#String algorithms`

and then merge your algorithms with rest of them, something like this 

```md
### C or C++
1. [Palindrome Check](c-or-cpp/palindrome.c)
2. [All subsequences](c-or-cpp/sequence.cpp)
3. [KMP String Searching](c-or-cpp/kmp.cpp)
4. [Rabin Karp String Searching](c-or-cpp/rabin-karp.cpp)
```